javascript/ql/src/Security/CWE-078/ShellCommandInjectionFromEnvironment.qhelp

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+<overview>
+	<p>
+
+		Dynamically constructing a shell command with values from the
+		local environment, such as absolute file paths, may inadvertently
```

Comment:
Saying "absolute file paths" instead of just "file paths" makes it seem like there is something special about them being absolute. Is there? If not, maybe just go with "file paths".